test/Conversion/tritongpu_to_llvm.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: triton-opt %s -split-input-file --convert-triton-gpu-to-llvm | FileCheck %s
+// RUN: triton-opt %s -split-input-file --convert-scf-to-cf --convert-triton-gpu-to-llvm | FileCheck %s
```

Comment:
is -convert-scf-to-cf still needed here?

Reply:
It's due to a legacy test that uses `scf`. Do you want me to rewrite the test in `cf`?

https://github.com/openai/triton/blob/9c200555d36d324847424595eca7a51d74f8c0ee/test/Conversion/tritongpu_to_llvm.mlir#L1140

Reply:
The test has been failed for a month but the failure is silent so we didn't notice

Reply:
> Do you want me to rewrite the test in cf?

Probably worth feeding the test through `--convert-scf-to-cf` manually and using the `cf` version in the test yep! Can be in a separate PR though; i'll merge this one.

> The test has been failed for a month but the failure is silent so we didn't notice
oh, that's not good!

Reply:
Sure, will update the test file.